tests/test_jump_to_definition_plugin.py

```diff
@@ -31,6 +30,13 @@ def wait_for_langserver_to_start(filetab):
     )
 
 
+# Don't know why this is sometimes needed
+def intense_super_update():
+    start = time.time()
+    while time.time() < start + 3:
+        get_main_window().update()
+
+
 @pytest.mark.skipif(
     sys.platform == "win32" and os.getenv("GITHUB_ACTIONS") == "true",
     reason="github actions windows very random",
```

Comment:
Once merged, should make separate pr removing these checks. Maybe not needed with `intense_super_update()`

Reply:
#586